js/description.js

```diff
@@ -29,8 +29,8 @@ const markupDescription = (value) => {
   const time = date.cloneNode(false);
   const commetn = date.cloneNode(false);
 
-  time.textContent = value.time;
-  commetn.textContent = value.commetn;
+  time.textContent = value ? value.time : "10 min read";
+  commetn.textContent = value ? value.commetn : 11;
```

Comment:
Why do you need to have default values, in case of error you don't need to render that item, instead you may show some notification to the user